doc/UserGuide.md

```diff
@@ -40,14 +40,16 @@ Format: `add NAME [p]p/PHONE_NUMBER [p]e/EMAIL [p]a/ADDRESS [t/TAG]...`
 > Words in `UPPER_CASE` are the parameters, items in `SQUARE_BRACKETS` are optional, 
 > items with `...` after them can have multiple instances. Order of parameters are fixed. 
 > 
+> The format for `ADDRESS` is as follows: `BLOCK`, `STREETNAME`, `UNIT`, `POSTALCODE`
```

Comment:
:+1:  for clearly mentioning the format and giving an appropriate example below.